### ISSUE TITLE
Delete buttons should have access to the todo id

### ISSUE THREAD

**Issue by LamarcusBarton** [gordonmzhu/questions#450]
when ever I load the code, I immediately get an error 
    Uncaught TyperError: cannot read property 'addEventListener' of null

this error is coming before any action is taken, or delete buttons clicked. 

<img width="1192" alt="Screen Shot 2020-06-13 at 1 41 12 PM" src="https://user-images.githubusercontent.com/66574853/84575927-f5d3bb80-ad7e-11ea-865e-07b54db66c3e.png">

can you point me in the right direction of what I am doing wrong, or should the last section be enclosed inside of an object? 

Code:
[https://gist.github.com/LamarcusBarton/b42d7b5aacbeab23ef15c1bdefb927a2](url)

**Comment by LamarcusBarton** [comment]
I wanted to add:

I deleted lines 150 to 153 and typed the code directly into the console after loading the page. It ran perfectly, so I can't understand why it doesn't work when loading the script containing the code.
<img width="987" alt="Screen Shot 2020-06-13 at 9 54 07 PM" src="https://user-images.githubusercontent.com/66574853/84582943-a8783e00-adc0-11ea-8181-7aa25dec7100.png">

**Comment by LamarcusBarton** [comment]
sorry for so many messages, I kept researching my the issue while I was waiting for you to respond. I finally came across a forum, saying the issue was the script was loading before the html elements. The solutions solution mentioned was placing the script in the element in the HTML code. I also decided to use the setTimeout() function you mentioned in prior videos. Both solutions worked, but still want to make sure I am on the write path
<img width="1348" alt="Screen Shot 2020-06-15 at 12 23 44 AM" src="https://user-images.githubusercontent.com/66574853/84618266-ac2ec200-ae9f-11ea-9afc-5f3f1385ecb3.png">
